web/gui/dashboard/src/domains/chart/reducer.ts

```diff
@@ -0,0 +1,43 @@
+import { createReducer } from "redux-act"
+
+import { fetchDataAction, updateChartDetailsAction } from "./actions"
+import { ChartState } from "./chart-types"
+
+export type StateT = {
+  [chartId: string]: ChartState
+}
+
+const initialState = {
+}
+export const initialSingleState = {
```

Comment:
what is initialSingleState?

Reply:
bad naming, i guess :) so if this is the state in that domain:
```
export type StateT = {
  [chartId: string]: ChartState
}
```
then `initialSingleState` is initial for `ChartState` type. It is exported so even if the state is empty, selectors will return `initialSingleState`